### PR TITLE
feat: big number config UI/UX changes

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: #9566

### Description:

This PR makes changes to the `Big Number` config.

#### Layout

image

- Smaller inputs
- `Label` and `Format` are a grid now and `label` takes more spans 

#### Comparison

image

- Compare to previous row is like a `Switch` heading
- "Compare by" is a SegmentedControl
- `Comparison Label` and `Format` are a grid now and `label` takes more spans 

#### Folder structure

Mimics the other configs, by having folders that are similar to the Tabs
```
ConfigTabs/
Layout/
Comparison/
```

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/VisualizationConfigs/BigNumberConfig/Comparison/index.tsx
Comment (magnew) [inline packages/frontend/src/components/VisualizationConfigs/BigNumberConfig/Comparison/index.tsx:1]: I'm not asking for a change here, but this folder layout got pretty heavy for such a small component. Do we really want 4 sub folders with index files? 

Its not really a problem, it just seems like a lot of structure. Unless you plan to add tests for all of these 😄 

Or maybe we decided to follow this pattern always even when there are a lot of single-file folders? I see you did it for the other configs as well.

**Review by magnew — APPROVED** [review]
This looks great. One small inconsistency you can fix or not is that on the layout tab the label input doesnt expand to fill the space of the format input and on the other tab it does. 

Layout
Screenshot 2024-04-03 at 11 35 24

Comparison
Screenshot 2024-04-03 at 11 35 17

**Comment by joaoviana** [comment]
> This looks great. One small inconsistency you can fix or not is that on the layout tab the label input doesnt expand to fill the space of the format input and on the other tab it does.
> 
> Layout Screenshot 2024-04-03 at 11 35 24
> 
> Comparison Screenshot 2024-04-03 at 11 35 17

Ahhh thanks for finding this @magnew !

**Comment by joaoviana** [comment]
Closing as the base can prevent us from pushing this code sooner

**Pull request closed without merging**